app/src/main/res/values/colors.xml

```diff
@@ -15,5 +15,6 @@
     <color name="category_3">#E5BA93</color>
     <color name="category_4">#9677C0</color>
     <color name="category_5">#4A8FB4</color>
+    <color name="text_delete_flash">#4C5763</color>
```

Comment:
I'm wondering the way we should name the colors, if just like the color name for example `dark_brown` or by the component where we are using it as you did. What do you think?

Reply:
perfect, so everything is standardized